indico/modules/events/management/forms.py

```diff
@@ -168,6 +168,13 @@ def validate_end_dt(self, field):
 
 class EventLocationForm(IndicoForm):
     location_data = IndicoLocationField(_('Location'), allow_location_inheritance=False)
+    own_map_url = URLField(_('Map URL'))
+
+    def __init__(self, *args, **kwargs):
+        self.event = kwargs.pop('event')
```

Comment:
no need to store it on `self` since it's not used anywhere else

Reply:
Also, I see no need to pop `event`. `IndicoForm` is not expecting it, but it was receiving it already anyways before this PR.

Reply:
I was just about to ask why you aren't popping it anymore ;) Anyway, passing it doesn't do any harm (usually we pop it unless we inherit from a custom base class that still needs it)